lib/waterline/adapter/types.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  supportsType: function(type) {
+    var adapter = this.connections[connName]._adapter;
+    return adapter.supportsType && adapter.supportsType(type);
```

Comment:
It would really depend on whether "default" types are definitely supported. If they are definitely supported, I suppose we can add an extra condition in front. What do you think?